package.json

```diff
@@ -70,8 +70,10 @@
     "request": "2.67.0",
     "serve-static": "1.10.0",
     "time-grunt": "1.2.1",
-    "ua-parser": "0.3.5",
-    "universal-analytics": "0.3.9"
+    "uap-core": "git://github.com/ua-parser/uap-core.git#1f509812",
+    "uap-ref-impl": "0.2.0",
+    "universal-analytics": "0.3.9",
+    "yamlparser": "0.0.2"
```

Comment:
Here's where we need to have Win 10 working too: https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/statsd.js#L6